### PR TITLE
Give special characters in scopes more emphasis

### LINKED ISSUE #904: [UI] Give special characters in scopes more emphasis
STR:

* Navigate to https://taskcluster-ui.herokuapp.com/auth/roles/hook-id%3Ahg-push%2Fautoland
* View the expanded scopes.
    
**Expected:** `assume:`, `*` and `<..>` substrings are given more emphasis. This could probably be done by making the special characters have a bold font.

**Actual:** special characters are not standing out.

Screen Shot 2019-06-24 at 5 59 40 PM

### PR DESCRIPTION
**Pull request by subhamskm**
Fixes #904

### CODE REVIEW

File: test/meta_test.js [hunk 1]
```diff
@@ -169,4 +169,26 @@ suite('Repo Meta Tests', function () {
       throw new Error(errors);
     }
   });
+  test("no references to tools.taskcluster.net in the repository", async function () {
```
Comment (arku) [inline test/meta_test.js]: @subhamskm Looks like this change is irrelevant to this PR and is instead related to #961

Can you please remove it?

Comment (arku) [reply test/meta_test.js]: Actually, you can drop the commits https://github.com/taskcluster/taskcluster/pull/974/commits/2c957e3475a695685ac7357aa607a04d5ef7f54c and https://github.com/taskcluster/taskcluster/pull/974/commits/0bc61e950f244bd550f9c080d8f0fb376c4d7160. Let me know if you need any help with it.

File: ui/src/components/RoleForm/index.jsx [hunk 2]
```diff
@@ -122,6 +123,19 @@ export default class RoleForm extends Component {
...
+        let res=[];
+        for (var i = 0; i < str.length; i ++) {
+          if(str.charAt(i).match(/[\/\.*:-]/g))
+            res[i] = <b>{str.charAt(i)}</b>;
+          else
+            res[i] = str.charAt(i);
+        }
+        console.log(res);
+        return <code>{res}</code>;
+      }
```
Comment (helfi92) [inline ui/src/components/RoleForm/index.jsx]:
```jsx
function formatScope(scope) {
  return scope.replace(
    /^assume:|\*$|<..>/g,
    match => `<strong>${match}</strong>`
  );
}
```

File: ui/src/components/RoleForm/index.jsx [hunk 3]
```diff
@@ -202,7 +216,7 @@ export default class RoleForm extends Component {
                           component={Link}
                           to={`/auth/scopes/${encodeURIComponent(scope)}`}
                           className={classes.listItemButton}>
-                          <ListItemText secondary={<code>{scope}</code>} />
+                          <ListItemText secondary={fun({scope})} />
```
Comment (helfi92) [inline ui/src/components/RoleForm/index.jsx]: Rewrite `ListItemText` as:

```jsx
<ListItemText
  disableTypography
  secondary={
    <Typography>
      <code
        // eslint-disable-next-line react/no-danger
        dangerouslySetInnerHTML={{
          __html: formatScope(scope),
        }}
      />
    </Typography>
  }
/>
```

**Comment by subhamskm** [comment]
Thanks @helfi92

**Pull request merged**